src/borg/repository.py

```diff
@@ -876,6 +886,19 @@ def delete(self, id, wait=True):
         self.compact[segment] += size
         self.segments.setdefault(segment, 0)
 
+    def async_wait(self):
+        """Wait until all outstanding async commands' responses have been received.
+        """
+
+    def async_response(self):
+        """Get one async result (only applies to remote repositories).
```

Comment:
µnitpick: docstring format should either be """oneline"""" or """\n multi\n line\n """, not mixed